views/PrestaShop/Admin/Common/Grid/grid_panel.html.twig

```diff
@@ -16,11 +16,16 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
+{% set displayName = true %}
+{% if (grid.view_options.display_name is defined) %}
+    {% set displayName = grid.view_options.display_name %}
+{% endif %}
+
 <div class="card js-grid-panel" id="{{ grid.id }}_grid_panel" data-hook-name="{{ grid.name|split(' ')|first }}">
   {% block grid_panel_header %}
-    {% if grid.name is not null or grid.actions.grid|length > 0 %}
+      {% if displayName is not sameas(false) or grid.actions.grid|length > 0 %}
       <div class="card-header js-grid-header">
-        {% if grid.name is not null %}
+          {% if displayName is not sameas(false) %}
```

Comment:
```suggestion
          {% if displayName is sameas(true) %}
```

Reply:
if display name is false, the name is not displayed. otherwise, whatever the value it will be displayed